src/uu/more/src/more.rs

```diff
@@ -138,9 +100,9 @@ fn reset_term(term: &mut redox_termios::Termios) {
     let _ = syscall::close(fd);
 }
 
-fn more(matches: getopts::Matches) {
-    let files = matches.free;
-    let mut f = File::open(files.first().unwrap()).unwrap();
+fn more(matches: ArgMatches) {
+    // FixME: currently panics but `more` should work with no arguments (ie, for piped input)
```

Comment:
are you planning to fix that ? thanks

Reply:
I tried to but it seems like the problem doesn't have anything to do with the way we handle arguments. the `FixME` comment was actually in the code in the first place I just moved it to the right place.